src/product-card.css

```diff
@@ -0,0 +1,90 @@
+.product-card {
+  width: 198px;
```

Comment:
Maybe try changing box-sizing to border-box, so that you can avoid such `magic numbers` in the code